fleet/s/dev/c/ruka/observability/fleet.yaml

```diff
@@ -0,0 +1,14 @@
+defaultNamespace: monitoring
```

Comment:
Lets install loki into a ns named `loki` to be consistent with the mimir deployment.